### ISSUE TITLE
openExternalView() & openExternalEdit() are broken

### ISSUE THREAD

**Issue by loic-hamdi** [QuisApp/flutter_contacts#91]
Hey,

On iOS physical device I experience an unexpected behavior with: 

```dart
await FlutterContacts.openExternalView(itemData.id);
```
or 
```dart
await FlutterContacts.openExternalEdit(itemData.id);
```

Console log: 
```
[PPT] Error creating the CFMessagePort needed to communicate with PPT.
```

The contact card opens up, however:
- Issue 1: In openExternalEdit mode it opens the view mode instead of edit mode.
- Issue 2 **(breaking)** : If you tap Edit the contact you cannot pop the modal anymore and the app is completely stuck on this screen.

Any idea how to fix this?

**Comment by starshipcoder** [comment]
For information, seems to work in this library
https://pub.dev/packages/contacts_service

**Comment by starshipcoder** [comment]
I have donen a PR : #104

**Comment by joachim-quis** [comment]
Should be fixed in version [1.1.7](https://pub.dev/packages/flutter_contacts/versions/1.1.7)
Thanks @starshipcoder

**Comment by aytunch** [comment]
@loic-hamdi can you see the edit mode opened by default when you do `openExternalEdit`? Or does the user have to press edit button manually?
